draft-ietf-quic-transport.md

```diff
@@ -5018,6 +5042,35 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+The Retire Prior To field is a request for the peer to retire all connection IDs
+with a sequence number less than the specified value.  This includes the initial
+and preferred_address transport parameter connection IDs.  The peer SHOULD
+immediately retire all the connection IDs.
```

Comment:
I would change this sentence to match the text before: `The peer SHOULD retire the corresponding connection IDs and send the corresponding RETIRE_CONNECTION_ID frame in a timely manner.` and also delete the next two paragraphs as they are redundant and contradict the "Detecting a Stateless Reset" section.

Reply:
To be clear, you suggesting I remove these two paragraphs?

> If a stateless reset is received in response to a retired connection ID, the
> reset SHOULD be ignored and the receiver SHOULD immediately switch to a
> connection ID that has not been retired if it has not done so already.
> 
> In order to prevent a heavily delayed packet with a retired connection ID from
> eliciting a stateless reset that closes the connection, the stateless reset
> token of the retired connection IDs SHOULD NOT be reused.

Reply:
Yes